tests/css_scss/scss.scss

```diff
@@ -1047,3 +1047,6 @@ $my-map: (
 label {
   @include margin-bottom-1\/3;
 }
+
+// should parse as custom-selector in scss
+@custom-selector     :--icon #id;
```

Comment:
It is very strange, because sass can't parser `@custom-selector     :--icon #id;` :smile: Maybe something wrong on parser side, remove last `;` from test to avoid parser problem for future updates and we can merge

Reply:
I missed it, first I comment `shouldn't` but I saw result is treat as `custom-selector`, so I change the comment

Reply:
should I remove the test from scss ?

Reply:
yep, let's remove this